gtest/msvc/Debug/gtest/gtest.log

```diff
@@ -0,0 +1,13 @@
+﻿Build started 15/04/2016 09:53:43 AM.
+     1>Project "C:\CallumProjects\PrintDiamond\gtest\msvc\gtest.vcxproj" on node 2 (Build target(s)).
+     1>ClCompile:
+         C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\CL.exe /c /I.. /I..\include /ZI /nologo /W3 /WX- /Od /Oy- /D WIN32 /D _DEBUG /D _LIB /D _MBCS /Gm- /EHsc /RTC1 /MTd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Fo"Debug\gtest\\" /Fd"Debug\gtest\vc120.pdb" /Gd /TP /analyze- /errorReport:prompt "..\src\gtest-all.cc"
+         gtest-all.cc
+       Lib:
+         C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\Lib.exe /OUT:"Debug\gtestd.lib" /NOLOGO "Debug\gtest\gtest-all.obj"
```

Comment:
Please add a .gitignore as we discussed earlier. :)
Gets rid of log and binary files - you only want source code (and other source material like icons, config files, ...) in your repository. Everything else is generated from that source and only bloats the repo.